src/DynamoCore/Graph/Nodes/NodeModel.cs

```diff
@@ -2546,7 +2546,15 @@ internal void RequestValueUpdate(EngineController engine)
             };
 
             var task = new UpdateRenderPackageAsyncTask(scheduler);
-            if (!task.Initialize(initParams)) return false;
+            try
+            {
+                if (!task.Initialize(initParams)) return false;
+            }
+            catch (ArgumentNullException e)
```

Comment:
which argument is null? if we know, we can just introduce a null check, can we?

Reply:
I have not been able to recreate this so don't know actually. It is one of the argument checks in Initialize that throws and the argument name is passed to the exception. Will we actually see what argument it is now with our recent updates to our exceptions logging? One other option would be to not throw at all in Initialize and just return false if anything is null.

Reply:
By looking at the `Initialize` method it appears that the `AstIdentifierForPreview` property of the nodemodel could be `null`. If so, could add a null check for it [here](https://github.com/DynamoDS/Dynamo/blob/01994440fdc5eb78f34ddb15a482b4bef4027859/src/DynamoCore/Scheduler/UpdateRenderPackageAsyncTask.cs#L96)?

Reply:
The call stack indicates an `ArgumentNullException`. My guess is that this happens at the very beginning of the initialize method where we check the arguments [here](https://github.com/DynamoDS/Dynamo/blob/01994440fdc5eb78f34ddb15a482b4bef4027859/src/DynamoCore/Scheduler/UpdateRenderPackageAsyncTask.cs#L77)?